sdk/spring/ci.yml

```diff
@@ -330,17 +330,17 @@ extends:
     ServiceDirectory: spring
     Artifacts: ${{ parameters.Artifacts }}
     AdditionalStagesAfterBuild:
-      - stage: TestCurrentSpringBootVersion
+      - stage: TestLatestVersion
         jobs:
           - template: ../../../../sdk/spring/spring-sdk-client-test.yml
             parameters:
               Artifacts: ${{ parameters.Artifacts }}
-              SpringBootVersion: 2.6.1
-              SpringCloudVersion: 2021.0.0
-      - stage: TestLastSpringBootVersion
+              SpringBootVersion: 2.6.3
+              SpringCloudVersion: 2021.0.1
+      - stage: TestCompatibleVersion
         jobs:
           - template: ../../../../sdk/spring/spring-sdk-client-test.yml
             parameters:
               Artifacts: ${{ parameters.Artifacts }}
-              SpringBootVersion: 2.5.7
-              SpringCloudVersion: 2020.0.4
+              SpringBootVersion: 2.6.2
+              SpringCloudVersion: 2021.0.0
```

Comment:
does this mean we will not test the compatibility with Spring Boot 2.5.x?

Reply:
It should be Spring Boot 2.5.x.
@Netyyyy could you please fix it?

1. Use the latest patch version of Spring Boot 2.5.x. Now it's 2.5.10. 
2. Find latest compatible spring cloud version. Now it's 2020.0.5. 

Refs: 
1. https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/
2. https://spring.io/projects/spring-cloud 
3. https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-dependencies/

Reply:
@chenrujun fixed in #27535, will change to 2.5.10 on next version update